site/package.json

```diff
@@ -30,12 +30,44 @@
     "gulp-rev-delete-original": "^0.2.3",
     "gulp-stylus": "^2.7.0",
     "gulp-uglify-es": "^1.0.4",
-    "gulp-webserver": "^0.9.1"
+    "gulp-webserver": "^0.9.1",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
+    "xo": "github:aslafy-z/xo#use-cosmiconfig"
   },
-  "author": "Swapnil Agarwal",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/swapagarwal/swag-for-dev/issues"
+  "scripts": {
+    "lint": "xo '**/*.js'",
+    "lint:fix": "xo --fix '**/*.js'",
+    "build": "gulp build"
+  },
+  "xo": {
+    "space": 4,
+    "plugins": [
+      "unicorn"
```

Comment:
XO already [includes the unicorn plugin by default](https://github.com/xojs/xo#highlights), and so there's no need to manually include it.

Reply:
It's needed as I disable a unicorn rules in the rules. Custom behavior like this one need a implicit import of the said plugin. Don't find the reference to it just now but it doesn't work without.

Reply:
From my testing of this branch locally without the `plugins` property specified in XO's config, linting seems to work properly (and the `unicorn/prefer-spread` rule stays disabled).
Is there anything specific that doesn't work for you when the `plugins` property isn't specified?

Reply:
Well it works as you said, I made the change!